docker/entrypoint.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -e
+container=docker
+export container
+
+if [ $# -eq 0 ]; then
+	echo >&2 'ERROR: No command specified. You probably want to run `journalctl -f`, or maybe `bash`?'
```

Comment:
It's not mandatory, but can we use double-quotes in lieu of backticks in this and other strings? Backticks in single-quoted strings shouldn't be a problem, but if something ever happens where we need a variable expansion and replace the single quotes with double quotes, the backtick-surrounded words will turn into subshells similar to if `$(...)` had been used.

Also, if you don't mind, please move the `>&2` to the end of the line here and elsewhere, and replace `echo` with `printf '%s\n'` here and in other locations. See https://www.kicksecure.com/wiki/Dev/bash#Safe_ways_to_print, `echo` is impossible to use safely unfortunately.

```suggestion
	printf '%s\n' 'ERROR: No command specified. You probably want to run "journalctl -f", or maybe "bash"?' >&2
```